src/server/rest/v1/service/LoggingService.ts

```diff
@@ -55,36 +55,43 @@ export default class LoggingService {
   }
 
   private static convertToCSV(req: Request, loggings: Log[], writeHeader = true): string {
-    let csv = '';
+    let headers = null;
     // Header
     if (writeHeader) {
-      csv = 'date' + Constants.CSV_SEPARATOR;
-      csv += 'time' + Constants.CSV_SEPARATOR;
-      csv += 'level' + Constants.CSV_SEPARATOR;
-      csv += 'type' + Constants.CSV_SEPARATOR;
-      csv += 'action' + Constants.CSV_SEPARATOR;
-      csv += 'message' + Constants.CSV_SEPARATOR;
-      csv += 'method' + Constants.CSV_SEPARATOR;
-      csv += 'module' + Constants.CSV_SEPARATOR;
-      csv += 'source' + Constants.CSV_SEPARATOR;
-      csv += 'host' + Constants.CSV_SEPARATOR;
-      csv += 'process' + Constants.CR_LF;
+      const headerArray = [
+        'date',
+        'time',
+        'level',
+        'type',
+        'action',
+        'message',
+        'method',
+        'module',
+        'source',
+        'host',
+        'process'
+      ]
```

Comment:
Why don't you use the same approach as in Charging Station and provide the join() here?

Reply:
when headers are only strings (no computing in the array) we can skip the join. Anyways I re-added it already to be consistent with other parts and not having surprise if one day we need changes